### ISSUE TITLE
Rename metrics to adhere to Prometheus metric naming conventions

### ISSUE THREAD

**Issue by stanhu** [larseen/pgbouncer_exporter#5]
Thanks for this project! We're looking at shipping a PgBouncer exporter in GitLab but noticed the metric names in this exporter don't adhere to Prometheus naming conventions: https://prometheus.io/docs/practices/naming/

https://github.com/spreaker/prometheus-pgbouncer-exporter#exported-metrics is an example of how the metrics should be named. 

The `avg` queries can probably also be omitted as well, since Prometheus can calculate averages over any time period needed.

**Comment by larseen** [comment]
Yeah, agree with the points made, unfortunately the project have not seen some work in some time.
You are welcome to make a Pull Request If you like!
